sycl/plugins/opencl/pi_opencl.hpp

```diff
@@ -0,0 +1,87 @@
+//==---------- pi_opencl.hpp - OpenCL Plugin -------------------------------==//
```

Comment:
why did you need to create the pi_opencl.hpp and move some things from .cpp?

Reply:
Just for consistency with the other plugins. Especially since the `_PI_OPENCL_PLUGIN_VERSION` I think belongs in a `hpp` file instead of a `cpp` file.

Reply:
So maybe just leave that one there and move everything else back to .cpp?

Reply:
OK sure

Reply:
thanks